src/components/calculator.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Let's not disable linter rules. These are here to help you write the best code.

Reply:
Noted with thanks